lib/driver/compilerTool.cpp

```diff
@@ -76,7 +76,9 @@ int Compiler([[maybe_unused]] int Argc,
   PO::Option<PO::Toggle> PropThreads(
       PO::Description("Enable Threads proposal"sv));
   PO::Option<PO::Toggle> PropAll(PO::Description("Enable all features"sv));
-
+  PO::Option<std::string> OptimizationLevel(
+      PO::Description("Optimization level, can be one of O1, O2, O3, Os, Oz"sv),
+      PO::MetaVar("OPTIM_LEVEL"sv), PO::DefaultValue(std::string("Oz")));
```

Comment:
I prefer to use:

```
--optimize=[0,1,2,3,s,z], and the default value is `2`
```
Because this option is not usually used in typical cases, we don't need to set the `MetaVar`. Also, the `MetaVar` is not very simple.